config/nav.yml

```
@@ -51,6 +51,7 @@ nav:
         - Uninstalling Knative: admin/install/uninstall.md
         # Serving config
         - Knative Serving configuration:
+          - Revision Garbage Collection: admin/serving/revision-gc.md
```

```suggestion
          - Revision garbage collection: admin/serving/revision-gc.md
```
titles should be sentence case unless it's a proper name of an object or something